debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -172,6 +172,20 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withWidth(Width.LONG)
             .withImportance(Importance.MEDIUM)
             .withDescription("");
+    
+    /**
+     * A comma-separated list of regular expressions that match fully-qualified names of columns to map a primary key.
```

Comment:
Interesting, what's the reasoning for regex? I rather was assuming this would be the list of comma-separated column name(s)?

Reply:
Well, I was not sure about that : according to @jpechane on gitter, he would prefer an option like "message.key.columns". So in this case, I think we need the "schema.table.column" regex in order to be able to find the right column for a given table.

Reply:
To be more precide, this acts like the "columnFilter"

Reply:
Ah, I get it now; we also need a link to the described table. So how about this: 

    message.key.columns=inventory.customers:pk1,pk2;*.purchaseorders:pk3,pk4

?

I.e. allow for mutiple (table-regex + column list) tuples?

Reply:
You are right, that is nice enhancement.

Reply:
done